src/main/java/io/vertx/core/http/impl/Http2ClientConnection.java

```diff
@@ -95,6 +103,9 @@ public void createStream(HttpClientRequestImpl req, Handler<AsyncResult<HttpClie
     synchronized (this) {
       try {
         Http2Connection conn = handler.connection();
+
```

Comment:
3 blank lines

Reply:
I'll address that after the other review because it will create unnecessary conflicts when rebasing http_improvements pr